pkg/controller/sriovnetworknodepolicy/sriovnetworknodepolicy_controller.go

```diff
@@ -149,9 +145,15 @@ func (r *ReconcileSriovNetworkNodePolicy) Reconcile(request reconcile.Request) (
 	if err = r.syncConfigDaemonSet(defaultPolicy); err != nil {
 		return reconcile.Result{}, err
 	}
-	// Render and sync Daemon objects
-	if err = r.syncPluginDaemonObjs(defaultPolicy, policyList); err != nil {
-		return reconcile.Result{}, err
+	if len(policyList.Items) > 1 {
```

Comment:
Is it an option to edit existing default policy CR and trigger provisioning of SRIOV vf?

Reply:
No, default policy shall contain no configuration at all. Adding this condition to make sure the configure map is only created when there is at least one policy defined by user.

Reply:
Do we need to include cni-daemonset and webhook in the same condition?

Reply:
cni-daemonset has already been included. For webhook, I think we want it to be deployed regardless. So it doesn't need to care if there is any user defined policy or not.